cmd/aro/mirror.go

```diff
@@ -130,19 +129,10 @@ func mirror(ctx context.Context, log *logrus.Entry) error {
 	// soverign clouds a separate mirror process mirrors from the public cloud
 	if env.Environment().Environment == azure.PublicCloud {
 		srcAcrGeneva := "linuxgeneva-microsoft" + acrDomainSuffix
-
-		// Mirror the versions that we have defined, as well as future versions
-		// for testing
 		mirrorImages := []string{
-			version.MdsdImage(srcAcrGeneva),
-			version.MdmImage(srcAcrGeneva),
```

Comment:
Why are we removing mirroring for these versions?  

I understand why we would add more, but why would we remove the ones that get set in cluster?

Reply:
The `distroless` gets dropped so we can't reuse the same pull spec :(

Reply:
ack - so on mirror, `/distroless` gets dropped when it's mirrored into our ACR.  Would it make sense to have a comment somewhere stating that?